src/hooks/useKnockoutExplorer.ts

```diff
@@ -131,6 +131,11 @@ async function configureFabric(): Promise<Explorer> {
             handleCachedDataMessage(data);
             break;
           }
+          case "setToolbarStatus": {
+            const cdeToolbar = document.getElementById("cde-commandBarContainer");
```

Comment:
It would be better to do it the React way and avoid manipulating the DOM. Can we set the status somewhere in some global context and inject the height as property into the react component instead?

Reply:
Resolved via https://github.com/Azure/cosmos-explorer/pull/1720/commits/2ae6392b7e11d8e2c72c3a012169b54f74238569, thanks!